extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -65,10 +68,19 @@ class AmpLightbox extends AMP.BaseElement {
       right: 0,
     });
 
+    if (this.isScrollable_) {
+      st.setStyles(this.element, {
+        overflowY: 'auto',
+        overflowX: 'hidden',
```

Comment:
@beck410 Very interesting! Looks like a paint issue as changing any HW-accelerated style like opacity or disabling/enabling -webkit-overflow-scrolling fixes it!

@muxin Let's make sure we reproduce this bug with new code first and then I suspect (based on some preliminary hacking) removing `-webkit-overflow-scrolling` on `close` and adding it again on open fixes the issue.